<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CmccForm</class>
 <widget class="QWidget" name="CmccForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CmccForm</string>
  </property>
  <property name="windowIcon">
   <iconset resource="lnnunetworkhelper.qrc">
    <normaloff>:/LnnuNetworkHelper/Network.ico</normaloff>:/LnnuNetworkHelper/Network.ico</iconset>
  </property>
  <widget class="QFrame" name="cmcc_info_frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>311</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="cmcc_logout_pb">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>登出</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>帐号:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLabel" name="cmcc_username_lb">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>本月已用:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLabel" name="cmcc_useMB_lb">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>套餐类型:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLabel" name="cmcc_useType_lb">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="cmcc_login_frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>311</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>292</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>用户名:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QComboBox" name="cmcc_username_cb">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="cmcc_deleteuser_pb">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>密码:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="cmcc_sms_pb">
       <property name="text">
        <string>短信密码</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>验证码:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="cmcc_vcode_le"/>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="cmcc_vcodepic_la">
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="cmcc_changevcode_pb">
       <property name="text">
        <string>换一张</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="cmcc_login_pb">
       <property name="text">
        <string>登陆</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QLabel" name="cmcc_state_la">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="3" alignment="Qt::AlignHCenter">
      <widget class="QCheckBox" name="cmcc_remember_cb">
       <property name="toolTip">
        <string>记住密码</string>
       </property>
       <property name="text">
        <string>记住</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="cmcc_password_le">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cmcc_login_pb</tabstop>
  <tabstop>cmcc_username_cb</tabstop>
  <tabstop>cmcc_password_le</tabstop>
  <tabstop>cmcc_vcode_le</tabstop>
  <tabstop>cmcc_deleteuser_pb</tabstop>
  <tabstop>cmcc_remember_cb</tabstop>
  <tabstop>cmcc_sms_pb</tabstop>
  <tabstop>cmcc_changevcode_pb</tabstop>
 </tabstops>
 <resources>
  <include location="lnnunetworkhelper.qrc"/>
 </resources>
 <connections/>
</ui>
